scRNA-seq-advanced/03-dataset_integration.Rmd

```diff
@@ -217,6 +218,9 @@ If we have annotations, they are a helpful "bonus" for assessing the integration
 
 ## Prepare the SCE list for integration
 
+![Single-cell roadmap: Merge](diagrams/roadmap_multi_merge.png)
```

Comment:
Whether we need the merge diagram here again is up to you. I don't think it is terrible to have it twice, but we probably don't need to, as we will have a second integration diagram later. Or if you take the suggestion to change the initial diagram, leaving this here makes sense!